modules/flight_interface/flight_interface.py

```diff
@@ -17,7 +19,9 @@ class FlightInterface:
     __create_key = object()
 
     @classmethod
-    def create(cls, address: str, timeout_home: float) -> "tuple[bool, FlightInterface | None]":
+    def create(
+        cls, address: str, timeout_home: float, first_waypoint_distance_tolerance: float
+    ) -> "tuple[bool, FlightInterface | None]":
```

Comment:
add the explanation of the additional inputs to the docstring, especially specifying. what the distance tolerance means - like the units